rusk/src/lib/schedule_loader.rs

```diff
@@ -0,0 +1,29 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use std::fs;
+use std::io;
+use std::path::Path;
+use rusk_vm::Schedule;
+
+
+#[derive(Debug)]
+pub enum Error {
+    ScheduleLoaderError(io::Error),
+    ScheduleDeserializationError(toml::de::Error)
+}
```

Comment:
I would have one type of error for `rusk` if possible, and make those part of it.